factory-ai-vision/EdgeSolution/modules/WebModule/ui/src/components/ConfigTaskPanel/ConfigTaskPanel.tsx

```diff
@@ -88,6 +88,13 @@ const useProjectData = (initialProjectData: ProjectData): [ProjectData, OnChange
         cloneProject.SVTCcameras = [];
         cloneProject.SVTCconfirmationThreshold = 0;
         cloneProject.SVTCparts = [];
+      } else if (key === 'fps' && value) {
+        // keep Decimal point 1
```

Comment:
I found that formatting the input value here will bring a bad user experience. I suggest we remove this formatting and just giving an error message in `CameraFPSOptions` just like we warn the user not to enter a value higher than the recommended fps. What do you think?

Reply:
ok